data/Chandraprakash-Darji.json

```diff
@@ -159,5 +159,9 @@
       "icon": "SiHashnode",
       "description": "People which don't watch youtube videos then can read the blogs"
     }
+  ],
+
+  "testimonials": [
+   "Deppu178"
```

Comment:
You probably be using an invalid username. Acc to LinkFree's commit history, it should be `Deepu178`.

```suggestion
   "Deepu178"
```

Reply:
Thanks for review